apps/web/src/app/tools/components/ToolBreadcrumbs.tsx

```diff
@@ -0,0 +1,53 @@
+import Link from "next/link";
```

Comment:
this shouldn't be called 'breadcrumbs'. Think about this. More like "ToolPageNavigation' or something